apps/src/aichat/views/aichatView.module.scss

```diff
@@ -12,4 +11,13 @@
   display: flex;
   flex-direction: row;
   gap: 2px;
+
+  .instructionsArea {
```

Comment:
I see the instructions area scrolls if needed. Why is it not necessary to use `overflow-y: scroll` here but needed for `conversationArea`?

Reply:
The instructions styling sets `overflow-y` internally ([here](https://github.com/code-dot-org/code-dot-org/blob/78e5be60cbc60e1c79b77a2e728339149f75537f/apps/src/lab2/views/components/instructions.module.scss#L104)) since just the text portion scrolls, not the whole component.